.github/workflows/codeflare-sdk-release-sync.yml

```
@@ -0,0 +1,80 @@
+      with:
+        ref: ${{ env.BRANCH_NAME }}
+
+    # Create a new branch
+    - name: Create a new branch
+      run: |
+        echo ${{ env.UPDATER_BRANCH }}
+        git checkout -b ${{ env.UPDATER_BRANCH }}
+        git push --set-upstream origin ${{ env.UPDATER_BRANCH }}
+
+    - name: Setup Python environment
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)